.github/ISSUE_TEMPLATE/report-a-bug.md

```diff
@@ -27,33 +32,43 @@ If you are reporting a bug, please follow the following steps:
 4.  If you are reporting a bug with commands or something else in-game,
     please include screenshots to help us diagnose the problem.
 
-5.  Make sure not to write between the arrows, as anything there will be
-    hidden.
-
-6.  Delete this line and all above lines before posting your issue!       -->
+-->
 
 ### Information
 
-**Full output of `/ess version`**:
-
+**Full output of `/ess version`:**
+<!--
+    Paste the output of `/ess version` inside the ```s, preferably copied
+    directly from the console.
+-->
 ```
-<!-- Run the command in your console, and copy and paste the whole output here -->
+
 ```
 
-**Server log**: <!-- Upload `logs/latest.log` to https://gist.github.com/, replace this with link -->
+**Server log:**
+<!--
+    Upload `logs/latest.log` to https://gist.github.com/ then paste the link
+    below this block.
+-->
+
+
+**EssentialsX config:**
+<!--
+    Upload `plugins/Essentials/config.yml` (and other config files if relevant)
+    to https://gist.github.com/ then paste the link below this block.
+-->
 
-**EssentialsX config**: <!-- Upload `plugins/Essentials/config.yml` to https://gist.github.com/, replace this with link -->
 
 ### Details
 
-**Description**  
-<!-- Replace this with a brief summary of the bug. -->
+**Description:**  
+<!-- What is the bug? Type a brief summary below this line. -->
```

Comment:
"brief summary" aren't summaries already brief?

Reply:
IMO while it might be redundant, it reinforces the fact we want a short and simple description here, not a massive unfocused paragraph.